### DISCUSSION TITLE
"Your Name" performed a deployment?

### DISCUSSION THREAD

**Discussion by sideup66** [gtxaspec/wz_mini_hacks#539] [Development]
Writing this as i just saw something weird on the codebase. The project has been dormant for a few months since October, I assume because were mostly stable. However. I recently saw an update today that merged a few PRs in...however the name is weird as it shows "Your Name" in the commit. Its not clickable nor searchable. Was that gtxaspec? Kinda concerned around the weirdness of the recent commit. If anyone can shed some light itll be appreciated. Thanks.

**Comment by gtxaspec** [comment]
Oops yeah that was me. I'll have to update my git settings lol

**Comment by sideup66** [comment]
Lol k, I got worried that the account was compromised lol
